openedx/core/djangoapps/course_views/course_views.py

```diff
@@ -0,0 +1,176 @@
+"""
+Tabs for courseware.
+"""
+from openedx.core.lib.api.plugins import PluginManager
+from xmodule.tabs import CourseTab, link_reverse_func
+
+_ = lambda text: text
+
+
+# Stevedore extension point namespaces
+COURSE_VIEW_TYPE_NAMESPACE = 'openedx.course_view_type'
+
+
+class CourseViewType(object):
+    """
+    Base class of all course view type plugins.
```

Comment:
Maybe this is my Java background, but to me this isn't the view itself (you don't create an instance of one). You are registering a type of view, that provides information as to how to create an instance of a view.

I agree that we should avoid using "view type" and "tab" interchangeably. I'll do a sweep through the docstrings.